dev/com.ibm.websphere.appserver.thirdparty.jsf-2.3/bnd.bnd

```diff
@@ -0,0 +1,52 @@
+#*******************************************************************************
+# Copyright (c) 2019 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+Fragment-Host: com.ibm.ws.org.apache.myfaces.2.3;bundle-version=1.0
+
+Bundle-Name: WebSphere JSF MyFaces Third Party API
+Bundle-SymbolicName: com.ibm.websphere.appserver.thirdparty.jsf-2.3
+Bundle-Description: WebSphere JSF 2.3 MyFaces Third Party API
+Implementation-Version: 2.3.3
+
+javac.source: 1.8
+javac.target: 1.8
+Require-Capability: osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=1.8))"
+
+# Don't export the org.apache.myfaces.buildtools package as that is only needed for compilation.  The jar that contains the 
+# org.apache.myfaces.buildtools package is javax.j2ee.jsf-2.2/lib/myfaces-builder-annotations-1.0.9.jar
+Export-Package: \
+  !org.apache.myfaces.buildtools.*, \
+  org.apache.myfaces.renderkit.html;version=${Implementation-Version}, \
+  org.apache.myfaces.shared.config;version=${Implementation-Version}, \
+  org.apache.myfaces.shared.renderkit;version=${Implementation-Version}, \
+  org.apache.myfaces.shared.renderkit.html;version=${Implementation-Version}, \
+  org.apache.myfaces.shared.renderkit.html.util;version=${Implementation-Version}
+
+# Import everything we need except for the below exclusions that will not be needed at runtime.
+Import-Package: \
+  !com.google.inject, \
+  !javax.ejb, \
+  !javax.persistence, \
+  !org.apache.tomcat, \
+  *
+
```

Comment:
Are these the imports actually required ?

Reply:
Yes they are - these are the same imports that have always been on the bundle.  I've just added them back because we're exporting o.a.m. packages from the bundle again.

Reply:
Let me ask this another way, now that we are not exposing for than 5 packages are these imports all necessary / appropriate?

Reply:
Good point, with the limited packages this bundle is OK without these import excludes - I'll remove them